tools/render-test/main.cpp

```diff
@@ -189,40 +206,39 @@ void renderFrameInner(
           0, 0, 0, 1 };
         memcpy(mappedData, kIdentity, sizeof(kIdentity));
 
-        renderer->unmap(gConstantBuffer);
+		m_renderer->unmap(m_constantBuffer);
```

Comment:
Seems like tabs slipping in again. Is this something where you have to switch VS editor settings when working on different projects?

The right solution is probably for us to check in an editorconfig (http://editorconfig.org/) file for the project. I believe VS is supposed to support that natively.